Demos cannot run on Windows 10

When trying to run on Windows 10 (cmd/conemu), I met the following error:
```
> go run unicode.go
panic: runtime error: index out of range

goroutine 1 [running]:
github.com/gdamore/tcell/terminfo.LookupTerminfo(0x0, 0x0, 0x0, 0x0, 0xc8)
        C:/Users/[username]/go/src/github.com/gdamore/tcell/terminfo/terminfo.go:766 +0x1292
github.com/gdamore/tcell.NewTerminfoScreen(0x1c06a8, 0x24, 0xc042067df0, 0x4101c1)
        C:/Users/[username]/go/src/github.com/gdamore/tcell/tscreen.go:40 +0x67
github.com/gdamore/tcell.NewScreen(0xc04207c601, 0x1240000000030, 0x110, 0x110)
        C:/Users/[username]/go/src/github.com/gdamore/tcell/screen.go:203 +0x2d
main.main()
        C:/Users/[username]/go/src/github.com/gdamore/tcell/_demos/unicode.go:73 +0x3e
exit status 2
```
Seems stuck at the line when creating the screen object
```go
s, e := tcell.NewScreen()
```


However WSL is fine (Ubuntu Bash/Conemu)
Grabbing using latest version of the library. Am I missing something?

--- a/terminfo/terminfo.go
+++ b/terminfo/terminfo.go
@@ -756,6 +756,12 @@ func loadFromFile(fname string, term string) (*Terminfo, error) {
 // one from the JSON file located in either $TCELLDB, $HOME/.tcelldb
 // or in this package's source directory as database.json).
 func LookupTerminfo(name string) (*Terminfo, error) {
+	if name == "" {
+		// else on windows: index out of bounds
+		// on the name[0] reference below
+		return nil, ErrTermNotFound
+	}
+
 	dblock.Lock()
 	t := terminfos[name]
 	dblock.Unlock()

